web/tests/test_teach.py

```
@@ -130,24 +130,25 @@ def test_post_authenticated_valid(self, mock_captcha):
         self.assertEqual(response.url, reverse("course_detail", kwargs={"slug": course.slug}))
 
     @patch("captcha.fields.CaptchaField.clean", return_value="PASSED")
-    def test_post_authenticated_email_mismatch(self, mock_captcha):
-        """Test POST for authenticated user with mismatched email"""
+    def test_form_for_authenticated_user(self, mock_captcha):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_form_for_authenticated_user`

Add return type annotation: `None`

(ANN201)